src/components/RaitingLabel/RaitingLabel.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
+import { CircularProgress, Typography, Box } from "@mui/material";
+
+const styles = {
+  top: 0,
+  left: 0,
+  bottom: 0,
+  right: 0,
+  position: "absolute",
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "center",
+};
+
+interface Props {
+  value: number;
+}
+
+export const CircularProgressWithLabel = ({ value }: Props) => (
```

Comment:
Your component should be named as the file. Rename value to a more understandable name.

Reply:
done